c++/measures/g4_iw.cpp

```diff
@@ -106,6 +109,8 @@ namespace cthyb {
     }
   }
 
+  // Define placeholders once and for all in types.hpp?
```

Comment:
I dislike defining global placeholders. By defining them here they are confined to this `.cpp` file and will not pollute the global namespace. This causes less surprises and allow me to skip using underscores to "protect" the place holder names. It is a matter of taste. / H